### PR TITLE
When company is a mention, link it

### PR DESCRIPTION
**Pull request by golles**
Please advise if this is the best way of doing this.

### CODE REVIEW

File: src/components/details/index.jsx [hunk 1]
```diff
@@ -42,6 +42,14 @@ const ListItem = ({ icon, title, value, link, skeleton = false }) => {
   );
 };
 
+const isCompanyMention = (company) => {
+  return company.startsWith('@') && !company.includes(' ');
```
Comment (golles) [inline src/components/details/index.jsx:46]: I'm not fully sure what the exact rules are for a mention, I think a mention should start with a `@` and cannot contain spaces.

File: src/components/details/index.jsx [hunk 2]
```diff
@@ -42,6 +42,14 @@ const ListItem = ({ icon, title, value, link, skeleton = false }) => {
...
+};
+
+const companyLink = (company) => {
+  return `https://github.com/${company.substring(1)}`;
```
Comment (golles) [inline src/components/details/index.jsx:50]: Strip first character (@)

File: src/components/details/index.jsx (hunk 1 above)
Comment (arifszn) [reply src/components/details/index.jsx:46]: Looks workable.

File: src/components/details/index.jsx (hunk 2 above)
Comment (arifszn) [reply src/components/details/index.jsx:50]: Right.

**Review by arifszn — APPROVED** [review]
Nice one! 

LGTM. Will you add anything? If not, we can merge it.

**Comment by golles** [comment]
> Nice one!
> 
> LGTM. Will you add anything? If not, we can merge it.

Thanks, this is all. You can merge it :-)

**Pull request merged**